cmd/metrics-server/app/options/options.go

```diff
@@ -54,16 +54,25 @@ type Options struct {
 
 func (o *Options) Validate() []error {
 	errors := o.KubeletClient.Validate()
-	if o.MetricResolution < 10*time.Second {
-		errors = append(errors, fmt.Errorf("metric-resolution should be a time duration at least 10s, but value %v provided", o.MetricResolution))
-	}
```

Comment:
Because `o.Logging.ValidateAndApply()` does not support multiple calls, so the `validate` function is wrapped separately.
so unit tests can be executed normally